Versions/Common/opie.lua

```diff
@@ -0,0 +1,36 @@
+local name, addon = ...
+local f = CreateFrame("Frame");
+local L = LibStub("AceLocale-3.0"):GetLocale("BeStride")
+
+-- Basedd off of Opie Rings\Default.lua file
+local function addBeStrideRingToOpie() 
+    local ringname = L["Bindings.Header"]
+    if not (OPie and OPie.CustomRings) then return end
+    local _, major = OPie.GetVersion()
+    if not (major and major == 4) then return end
+
+    OPie.CustomRings:AddDefaultRing(ringname, {
+        {"macrotext", "/click BeStride_ABRegularMount", icon="Interface/Icons/inv_misc_summerfest_brazierorange", fastClick=true},
+        {"macrotext", "/click BeStride_ABPassengerMount", icon="Interface/Icons/inv_misc_stonedragonorange"},
+        {"macrotext", "/click BeStride_ABGroundMount", icon="Interface/Icons/misc_arrowdown"},
+        {"macrotext", "/click BeStride_ABRepairMount", icon="Interface/Icons/trade_blacksmithing"},
+        name=ringname, hotkey="SHIFT-SPACE", _u="OPCBR"
```

Comment:
Admittedly, Shift-Space won't work because it's prebound to the "Toggle Screen Score" for battlegrounds. This is just a default and any use can/should override it but idk what a decent default would be for people. Should update it before merging though or OPie gives this warning (Luckily OPie is smart enough to not try overriding it)

![image](https://user-images.githubusercontent.com/20718430/205437620-c70c6e39-ed13-47af-9347-bfa76877c93f.png)